esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -312,20 +315,13 @@ DATABASE_NAME = '$DBNAME'
 DATABASE_HOST = 'localhost'
 DATABASE_PORT = '5432'
 
+VARNISH_HOST = 'localhost'
+VARNISH_PORT = '80'
+
 from database_settings import *
 
 MIDDLEWARE_LOCAL = []
-
-# E-mails for contact form
-email_choices = (
-    ('general', 'General Inquiries'),
-    ('web',     'Web Site Problems'),
-    )
-# Corresponding email addresses                                                                                                                                 
-email_addresses = {
-    'general': '$GROUPEMAIL',
-    'web':     '$SITENAME-websupport@learningu.org',
-    }
+SECRET_KEY = '`${CURDIR}/random_password.sh`'
```

Comment:
I didn't realize it was going to be significant additional work, so that's fine -- we can just leave that issue open.  I do think it would be a good idea to move to a single template at some point, but no reason to hold up this PR if it's a significant additional thing.